File: lang/python/test/fun/vac/data_structures/LinkedQueueTest.py
from fun.vac.data_structures.LinkedQueue import LinkedQueue
from fun.vac.util import TestRunner

def testLinkedQueue() -> bool:
    size = 8192

    queue = LinkedQueue()

    for i in range(1, size + 1):
        queue.enqueue(i)

    if queue.size() != size:
        return False

    for i in range(1, size + 1):
        if queue.peek() != i:
            return False
        queue.dequeue()

    return queue.isEmpty()

def main() -> None:
    TestRunner.parseTest(testLinkedQueue())

if __name__ == "__main__":
    main()

File: lang/php/test/fun/vac/data_structures/LinkedQueueTest.php
<?php
    require_once "fun/vac/data_structures/LinkedQueue.php";
    require_once "fun/vac/util/TestRunner.php";

    use fun\vac\data_structures\LinkedQueue\LinkedQueue;
    use fun\vac\util\TestRunner;

    function testLinkedQueue(): bool {
        $size = 8192;

        $queue = new LinkedQueue();

        for ($i = 1; $i <= $size; $i++) {
            $queue->enqueue($i);
        }

        if ($queue->size() != $size) {
            return false;
        }

        for ($i = 1; $i <= $size; $i++) {
            if ($queue->peek() != $i) {
                return false;
            }
            $queue->dequeue();
        }

        return $queue->isEmpty();
    }

    if (count(debug_backtrace()) == 0) {
        TestRunner\parseTest(testLinkedQueue());
    }
?>
